pkg/bandwidth/bandwidth.go

```diff
@@ -98,8 +98,12 @@ func InitBandwidthManager() {
 		{"net.core.netdev_max_backlog", "1000"},
 		{"net.core.somaxconn", "4096"},
 		{"net.core.default_qdisc", "fq"},
-		{"net.ipv4.tcp_congestion_control", "bbr"},
 		{"net.ipv4.tcp_max_syn_backlog", "4096"},
+		// Temporary disable setting bbr for now until we have a
+		// kernel fix for pacing out of Pods as described in #15324.
+		// Then, kernels with the fix can use bbr, and others cubic.
+		//{"net.ipv4.tcp_congestion_control", "bbr"},
```

Comment:
Nit: I'd remove that, it's easy to remember and we already have the commit history anyway. Probably not worth repushing if it's the only comment.

Reply:
Yeah, agree, just wanted to have a short explanation in the code on why we had to temporarily disable it.

Reply:
Ah, got it :) Once CI is done, I'll remove that single commented line.